### PR TITLE
Feature/add new plugins

### PR DESCRIPTION
**Pull request by peavers**
Adds requested plugins and preforms minor package maintenance.

### CODE REVIEW

**Review by bmbferreira — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -84,6 +86,7 @@ def installAll(toolVersion: String) =
      |npm install -g eslint-plugin-backbone@2.1.1 &&
      |npm install -g eslint-plugin-chai-expect@2.0.1 &&
      |npm install -g eslint-plugin-chai-friendly@0.4.1 &&
+     |npm install -g eslint-plugin-compat@3.1.1 &&
```
Comment (bmbferreira) [inline build.sbt:89]: don't forget to add this to ESLint.scala

File: build.sbt [hunk 2]
```diff
@@ -60,6 +60,7 @@ def installAll(toolVersion: String) =
      |npm install -g eslint-config-es6@1.10.3 &&
      |npm install -g eslint-config-gatsby-standard@2.2.0 &&
      |npm install -g eslint-config-google@0.13.0 &&
+     |npm install -g eslint-config-kentcdodds@14.3.2 &&
```
Comment (bmbferreira) [inline build.sbt:63]: don't forget to add this to ESLint.scala

**Review by bmbferreira — CHANGES_REQUESTED** [review]
All the new plugins should be added to the config file that we will use when none is provided. This file is generated on the ESLint.scala file

File: build.sbt (hunk 2 above)
Comment (peavers) [reply build.sbt:63]: This one being only a config rather than a plugin doesn't need to be added if I understood the code correctly?

Comment (bmbferreira) [reply build.sbt:63]: ops! you're right. this comment was only valid for compat plugin!

**Review by bmbferreira — APPROVED** [review]

**Review by DReigada — APPROVED** [review]

**Pull request merged**